src/renderers/shared/reconciler/ReactCompositeComponent.js

```diff
@@ -521,6 +522,7 @@ var ReactCompositeComponentMixin = {
       }
       return assign({}, currentContext, childContext);
     }
+    ReactCurrentOwner.processingChildContext = null;
```

Comment:
Bad to have an API that is null or true.  If it is a boolean api, all values should be booleans.